packages/core/src/interfaces/charts.ts

```diff
@@ -306,21 +312,12 @@ export interface HistogramChartOptions extends AxisChartOptions {
 /**
  * options specific to line charts
  */
-export interface LineChartOptions extends ScatterChartOptions {
-	/**
-	 * options for the curve of the line
-	 */
-	curve?: string | { name: string }
-}
+export interface LineChartOptions extends ScatterChartOptions {}
 
 /**
  * options specific to area charts
  */
 export interface AreaChartOptions extends AxisChartOptions {
-	/**
-	 * options for the curve of the line
-	 */
-	curve?: string | { name: string }
```

Comment:
Moved to AxisChartOptions since it's needed for combo and line chart demos as well.